### PR TITLE
html page for roster

### PR DESCRIPTION
**Pull request by RyanSalamone**

### CODE REVIEW

File: public/roster.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
+  <title>SunnySide Middle School</title>
+
+  <!-- CSS  -->
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (BenHCalvert) [inline public/roster.html]: can you replace with `
  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
  <link href= "./styles/styles.css" type="text/css" rel="stylesheet" media="screen,projection"/>
  <link href="./styles/materialize.css" type="text/css" rel="stylesheet" media="screen,projection"/>`

File: public/roster.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
+  <title>SunnySide Middle School</title>
+
+  <!-- CSS  -->
```
Comment (BenHCalvert) [inline public/roster.html]: `
  <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
  <script src="./js/materialize.js"></script>
  <script src= "./js/init.js"></script>`

**Comment by RyanSalamone** [comment]
approved

File: public/roster.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href= "../public\styles\styles.css" type="text/css" rel="stylesheet" media="screen,projection"/>
```
Comment (ferb00000100) [inline public/roster.html]: needs to be changed to "./styles/styles.css"

File: public/roster.html [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+  <link href="../public\styles\materialize.css" type="text/css" rel="stylesheet" media="screen,projection"/>
```
Comment (ferb00000100) [inline public/roster.html]: needs to be changed to "./styles/materialize.css"

File: public/roster.html [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    </div>
+    <div class="footer-copyright">
+     
+    </div>
+  </footer>
+
+
+  <!--  Scripts-->
+  <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
+  <script src="../public\js\materialize.js"></script>
```
Comment (ferb00000100) [inline public/roster.html]: needs to be changed to "./js/materialize.js"

File: public/roster.html [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+  <script src= "../public\js\init.js"></script>
```
Comment (ferb00000100) [inline public/roster.html]: needs to be changes to "./js/init.js"

**Comment by BenHCalvert** [comment]
I couldn't figure out how to checkout this branch and update the file paths so I made a new branch and created the roster page there.

**Pull request closed without merging**